runtime-modules/blog/src/lib.rs

```diff
@@ -578,36 +637,117 @@ decl_module! {
             Ok(())
         }
 
+        /// Remove reply from storage
+        ///
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (1)` doesn't depends on the state or parameters
+        /// - DB:
+        ///    - O(1) doesn't depend on the state or parameters
+        /// # </weight>
+        #[weight = BlogWeightInfo::<T, I>::delete_reply()]
+        pub fn delete_replies(
+            origin,
+            participant_id: ParticipantId<T>,
+            replies: Vec<(PostId, T::ReplyId, bool)>,
```

Comment:
Could we use here some struct with named and commented fields instead of the tuple?